src/library/coverart.cpp

```diff
@@ -67,6 +81,48 @@ QDebug operator<<(QDebug dbg, const CoverInfoRelative& infoRelative) {
             .arg(coverInfoRelativeToString(infoRelative));
 }
 
+QImage CoverInfo::loadImage(
+        const SecurityTokenPointer& pTrackLocationToken) const {
+    if (type == CoverInfo::METADATA) {
+        if (trackLocation.isEmpty()) {
+            qDebug() << "CoverArtUtils::loadCover METADATA cover with empty trackLocation.";
```

Comment:
Can this happen or should we guard it with an assertion?

Reply:
Probably yes. I've never seen those log messages. Done